cli/esbuild.config.mjs

```diff
@@ -136,7 +136,7 @@ const __dirname = __dirname__(__filename);
 		"pkce-challenge",
 		"pretty-bytes",
 		"proper-lockfile",
-		"ps-tree",
```

Comment:
`ps-tree` was causing #2970 - refactored to `ps-list`, which needs to be marked as external